google/oauth2/service_account.py

```diff
@@ -306,10 +319,11 @@ def _make_authorization_grant_assertion(self):
     @_helpers.copy_docstring(credentials.Credentials)
     def refresh(self, request):
         assertion = self._make_authorization_grant_assertion()
-        access_token, expiry, _ = _client.jwt_grant(
+        access_token, expiry, refresh_grant_response = _client.jwt_grant(
```

Comment:
that's a good point. Hmm. I guess I wanted to use the same names that the `jwt_grant` function used for those values, then show mapping those values to properties (which have slightly different names). But I'm good to change it if you want.